### ISSUE TITLE
[bitnami/external-dns] pdns apiPort should be a number instead of a string

### ISSUE THREAD

**Issue by rgl** [bitnami/charts#19274]
### Name and Version

bitnami/external-dns 6.25.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Install the external-dns with an `pdns.apiPort` that is a number.

### Are you using any custom parameters or values?

```yaml
...
      pdns:
        apiUrl: http://pdns-api
        apiPort: 8081
        secretName: pdns-api
```

### What is the expected behavior?

The chart should deploy without errors.

The `apiPort` should be declared as a number. Actually, I think that all ports should be numbers, in the `values.yaml` file and in the schema.

### What do you see instead?

From an Ansible task output, we can see the helm error:

```
TASK [kind : Install external-dns] *********************************************
fatal: [dm1]: FAILED! => changed=false 
  command: /usr/local/bin/helm diff upgrade kube-external-dns bitnami/external-dns --version=6.25.1 --reset-values -f=/tmp/tmp7gh8ek80.yml
  msg: |-
    Failure when executing Helm command. Exited 1.
    stdout:
    stderr: Error: Failed to render chart: exit status 1: Error: values don't meet the specifications of the schema(s) in the following chart(s):
    external-dns:
    - pdns.apiPort: Invalid type. Expected: string, given: integer
  
  
    Error: plugin "diff" exited with error
  stderr: |-
    Error: Failed to render chart: exit status 1: Error: values don't meet the specifications of the schema(s) in the following chart(s):
    external-dns:
    - pdns.apiPort: Invalid type. Expected: string, given: integer
  
  
    Error: plugin "diff" exited with error
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

### Additional information

This started to happen after https://github.com/bitnami/charts/pull/19190 was merged.

**Comment by rgl** [comment]
It's at https://github.com/bitnami/charts/pull/19287. Not entirely sure it will work. Please review it.

**Comment by javsalgar** [comment]
Thank you so much for the PR! The team will take a look

**Comment by fmulero** [comment]
Thanks a lot @rgl

We have just reverted the changes from https://github.com/bitnami/charts/pull/19194 due to similar issues in other charts. In a few hours a new version will be released removing the `schema` validation. We have created an internal task to improve the [readme-generator-for-helm tool](https://github.com/bitnami-labs/readme-generator-for-helm) and adapt the `values.yaml` file definition to generate the schemas according the needs of each chart.  Sorry for the inconveniences.
